Utils/fetchUtils.js

```diff
@@ -45,3 +45,13 @@ export const fetchCommentsByReviewId = (review_id) => {
       console.log(err);
     });
 };
+
+export const incrementVotes = (review_id, votesData, SetIsError) => {
+  return NcGamesAPI.patch(`/api/reviews/${review_id}`, votesData)
+    .then((res) => {
+      res.data;
+    })
+    .catch((err) => {
```

Comment:
We don't want a .catch in our utils. We want this instead in the component where this util is being invoked